src/engine/client/cl_input.cpp

```diff
@@ -1342,3 +1342,16 @@ void CL_ClearKeys()
 
 	memset( kb, 0, sizeof( kb ) );
 }
+
+void CL_ClearInput()
+{
+	int i;
+
+	for ( i = 0; i < USERCMD_BUTTONS; ++i )
+	{
+		if ( registeredButtonCommands[ i ] )
+		{
+			Z_Free( registeredButtonCommands[i] );
```

Comment:
Other places that free `registeredButtonCommands[ i ]`, also assign it to **nullptr**, perhaps this should do the same.